heartbeat/aws-vpc-move-ip

```diff
@@ -154,6 +156,15 @@ Enable enhanced monitoring using AWS API calls to check route table entry
 <shortdesc lang="en">Enhanced Monitoring</shortdesc>
 <content type="boolean" default="${OCF_RESKEY_monapi_default}" />
 </parameter>
+
+<parameter name="eni_id_lookup" required="0">
```

Comment:
I'd like to see a lookup_type parameter (doesn't have to use the name "lookup_type", just an example) that expects a string instead of a boolean. Its default value can be "interface_id" or something like that, and it can accept "eni_id" as a configured value. It's semantically more clear than asking "is this an eni_id_lookup or not?".

It's also extensible. That is, if for some reason we ever need a third lookup type, then using a boolean-based approach would get clunky quickly. It would require a new boolean parameter for each new lookup type, and you would have to add logic to decide what should be done if two of them are enabled at the same time (a config mistake). A string-based approach, on the other hand, could simply start accepting the new value.

Reply:
There is currently not more useful information coming back from the API call. But I have changed the code, still future additions would not integrate that easy because you will also need metadata information for your check. But I hope this makes it more flexible.

Reply:
Thanks :)